imagetool/src/main/java/com/oracle/weblogic/imagetool/cachestore/FileCacheStore.java

```diff
@@ -99,19 +107,22 @@ public String deleteFromCache(String key) {
     }
 
     private boolean persistToDisk() {
+        logger.entering();
         boolean retVal = true;
         synchronized (properties) {
             try (FileOutputStream outputStream = new FileOutputStream(metadataPath)) {
                 properties.store(outputStream, "changed on:" + LocalDateTime.now());
             } catch (IOException e) {
                 retVal = false;
-                e.printStackTrace();
+                logger.fine("Could not persist cache file", e);
```

Comment:
Not throwing exception ?

Reply:
It didn't before, and I didn't change it.  But, I had the same concern initially.  Instead of throwing an exception, it returns "false", which allows the calling routine to create an error response.  The return of "false" causes the tool to exit, so I think this is okay.

Reply:
throws exception now